packages/batch/src/errors.ts

```diff
@@ -1,49 +1,58 @@
+import { EventType } from './constants';
+
 /**
- * Base error type for batch processing
- * All errors thrown by major failures extend this base class
+ * Base error thrown by the Batch Processing utility
  */
-class BaseBatchProcessingError extends Error {
-  public childErrors: Error[];
-
-  public msg: string;
-
-  public constructor(msg: string, childErrors: Error[]) {
-    super(msg);
-    this.msg = msg;
-    this.childErrors = childErrors;
+class BatchProcessingError extends Error {
```

Comment:
Would we want the error thrown in the `BatchProcessor` [here](https://github.com/aws-powertools/powertools-lambda-typescript/blob/3ad0f5cffad7eb03fc2ac6deb78a29a1fad1a7fa/packages/batch/src/BatchProcessor.ts#L26C2-L26C2) to be as this `BatchProcessingError` instead of just `Error`, now that we have this?

Reply:
Indeed, very good catch. Updated in the latest commit.